node/executor/src/lib.rs

```diff
@@ -88,17 +88,15 @@ mod tests {
 	}
 
 	/// Default transfer fee
-	fn transfer_fee<E: Encode>(extrinsic: &E) -> Balance {
+	fn transfer_fee<E: Encode>(extrinsic: &E, fm: Fixed64) -> Balance {
```

Comment:
fm stands for fee multiplier ? I would maybe write this in the doc of the function and rename to a more sensible name

Reply:
I mean just for this one because it is in function argument, later use of fm are fine